pkg/asset/machines/openstack/machines.go

```diff
@@ -43,14 +43,14 @@ func Machines(clusterID string, config *types.InstallConfig, pool *types.Machine
 	if pool.Replicas != nil {
 		total = *pool.Replicas
 	}
-	var machines []clusterapi.Machine
+	var machines []machineapi.Machine
 	for idx := int64(0); idx < total; idx++ {
 		az := ""
 		provider, err := provider(clusterID, clustername, platform, mpool, osImage, az, role, userDataSecret)
 		if err != nil {
 			return nil, errors.Wrap(err, "failed to create provider")
 		}
-		machine := clusterapi.Machine{
+		machine := machineapi.Machine{
 			TypeMeta: metav1.TypeMeta{
 				APIVersion: "cluster.k8s.io/v1alpha1",
```

Comment:
you'll need to update the APIVersion here

Reply:
and namespace to `openshift-machine-api`

Reply:
good catch, thanks

Reply:
There's a separate patch for the `openshift-machine-api` change